__tests__/test.js

```diff
@@ -66,3 +69,27 @@ describe("test DB for requests", () => {
       });
   });
 });
+
+describe("GET /api/articles ", () => {
+  test("/api/articles returns all article in order of date", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((res) => {
+        expect(res.status).toBe(200);
+        expect(res.body.length).toBeGreaterThan(0);
+        expect(res.body).toBeSorted("comments.created_at");
+        expect(res.body).toBeSorted({ descending: true });
+        res.body.forEach((topic) => {
+          expect(topic).toHaveProperty("title");
+          expect(topic).toHaveProperty("article_id");
+          expect(topic).toHaveProperty("author");
+          expect(topic).not.toHaveProperty("body");
+          expect(topic).toHaveProperty("topic");
+          expect(topic).toHaveProperty("created_at");
+          expect(topic).toHaveProperty("votes");
+          expect(topic).toHaveProperty("article_img_url");
+          expect(topic).toHaveProperty("comment_count");
+        });
```

Comment:
the articles array should be coming back on a key that describes what it is e.g. `res.body.articles` should be where the data lives not directly on `res.body`

Reply:
This goes for all your endpoints so far, you should not be sending data directly back on the body